persistence-api/src/main/java/io/stargate/db/cdc/CDCServiceImpl.java

```diff
@@ -47,8 +47,19 @@
   }
 
   public CDCServiceImpl(CDCProducer producer, MetricRegistry registry) {
```

Comment:
Unrelated: should CDC service _implementation_ be in the _API_ bundle?

Reply:
Good question... once we have a clear view on how to integrate it with the persistence API, we can have a clear view.

We can discuss this sort of topics on the CDC basic PR #159